src/App.svelte

```diff
@@ -59,6 +63,9 @@
   });
 
   onMount(async () => {
+    await onOpenUrl(handleDeepLinkHandler);
```

Comment:
Can we put a platform check here. In MacOS it might trigger twice. In case of macOs it should be onOpenUrl or else listen.

Reply:
resolved